pkg/crc/machine/bundle/copier.go

```diff
@@ -61,41 +61,26 @@ func (copier *Copier) CachedPath() string {
 func (copier *Copier) CopyPrivateSSHKey(srcPath string) error {
 	sshKeyFileName := filepath.Base(copier.srcBundle.GetSSHKeyPath())
 	destPath := copier.resolvePath(sshKeyFileName)
-	if err := crcos.CopyFileContents(srcPath, destPath, 0400); err != nil {
-		return err
-	}
-	// copier.srcBundle.ClusterInfo.SSHPrivateKeyFile = sshKeyFileName
-	return nil
+	return crcos.CopyFileContents(srcPath, destPath, 0400)
```

Comment:
Ah these are in `Add e2e test for bundle creation command` rather than in the previous commit, I missed these when I looked again at the PR.

Reply:
@cfergeau fixed.